apps/web/src/components/WalletConnect/ProjectInfoCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { Avatar, Box, Card, Flex, Icon, Link, Text } from "@chakra-ui/react";
+import { type SignClientTypes } from "@walletconnect/types";
+
+import { PencilIcon } from "../../assets/icons";
+
+type Props = {
+  metadata: SignClientTypes.Metadata;
+  intention?: string;
+};
+
+/**
+ * dApp project info card. Contains verification info to help user decide if the dApp is safe to connect.
+ */
+export const ProjectInfoCard = ({ metadata, intention }: Props) => {
+  const { icons, name, url } = metadata;
+
+  return (
+    <Box textAlign="center">
+      <Box>
+        <Avatar marginX="auto" size="lg" src={icons[0]} />
+      </Box>
+      <Box marginTop="16px">
+        <Card data-testid="session-info-card-text">
+          <Text as="span" fontWeight="bold">
+            {name}
+          </Text>{" "}
```

Comment:
if this blank space is needed then please put it inside of either of the text tags